packages/loader/container-loader/src/loader.ts

```diff
@@ -428,3 +493,52 @@ export class Loader implements IHostLoader {
 		);
 	}
 }
+
+/**
+ * {@inheritDoc @fluidframework/container-definitions#ILoader.resolve}
+ * @legacy
+ * @alpha
+ */
+export async function resolveContainer(
```

Comment:
"resolve" has always been unintuitive to customers.  Let's rename to loadContainer I think.

Reply:
renamed to loadExistingContainer